go.mod

```diff
@@ -1,8 +1,10 @@
 module github.com/ratify-project/ratify-verifier-plugin
 
-go 1.21
+go 1.22.5
 
-require github.com/deislabs/ratify v1.2.1
+toolchain go1.23.1
```

Comment:
is it necessary?

Reply:
Checking

Reply:
Golang introduced go toolchain since go v1.21: https://tip.golang.org/doc/toolchain#version. Some of the notation-go dependencies have started to use it, for example [golang/mod](https://github.com/golang/mod).